pkg/operator/controller/vulnerabilityreport.go

```diff
@@ -108,6 +108,18 @@ func (r *VulnerabilityReportReconciler) reconcileWorkload(workloadKind kube.Kind
 			}
 		}
 
+		if r.Config.VulnerabilityScannerScanOnlyCurrentRevisions && workloadKind == kube.KindReplicaSet {
+			controller := metav1.GetControllerOf(workloadObj)
+			activeReplicaSet, err := r.IsActiveReplicaSet(ctx, workloadObj, controller)
+			if err != nil {
+				return ctrl.Result{}, err
```

Comment:
To give context when we see this error in logs:
```suggestion
				return ctrl.Result{}, fmt.Errorf("failed checking current revision: %w", err)
```